docs/spark-writes.md

```diff
@@ -203,6 +203,28 @@ WHERE EXISTS (SELECT oid FROM prod.db.returned_orders WHERE t1.oid = oid)
 
 For more complex row-level updates based on incoming data, see the section on `MERGE INTO`.
 
+### `Writing to Branches`
+Branch writes can be performed via SQL by providing a branch identifier, `branch_yourBranch` in the operation.
```

Comment:
we probably want to talk about both writing use identifier and writing WAP branch cases

Reply:
Done!